README.md

```diff
@@ -115,6 +110,39 @@ clevertap.profile.push({
 })
 ```
 
+### On User Login
+
+Enrich the user profile with information, such as the user’s name or email, by calling `clevertap.onUserLogin`.
```

Comment:
If multiple users on the same device use your app, you can use the onUserLogin method to assign them each a unique profile to track them separately.